### DISCUSSION TITLE
I always have to click the page each time in the extension to make it black and it gets on my nerves. Is there any way this extension automatically darkens all the websites I click on?

### DISCUSSION THREAD

**Discussion by 22xx06** [darkreader/darkreader#9146] [Q&A]
All said in the headline.

**Comment by Gusted** [comment]
> All said in the headline.

And the headline is quite confusing. Use it as a title and not to describe the issue you're having.
Dark Reader already by default does the described behavior, otherwise it's a bug.

**Reply by 22xx06** [reply]
Sorry, I am bad at English and I am usually not on such forums.

As my confusing headline said, I always have to press the extension button on the top right and activate the extension on the respective page (Chrome). But this turns out to be quite annoying, which is why I would like a function or a fix that directly darkens all web pages.

**Reply by Gusted** [reply]
Okay so, it's likely that you have enabled this behavior to only allow dark reader when you click on the extension.
![image](https://user-images.githubusercontent.com/25481501/177013220-8ffbfa88-f991-454e-bc1e-57e284a8f391.png)

To change this to it's normal behavior(enabling the extension without having to click), you will have to go to `chrome://extensions` and then click on the "Details" button for the Dark Reader card:
![image](https://user-images.githubusercontent.com/25481501/177013241-bd12f019-c336-4093-93a2-154b274e52c5.png)

And then proceed to change the "On click" value to "On all sites".

**Reply by 22xx06** [reply]
Nope, this isnt the case, I already have it on "On all sites".

**Reply by Gusted** [reply]
Oh that's weird, we never got such issue before, it could be a browser issue. Dark reader doesn't have any code to defer injecting it's code into sites.
